tools/tidy

```diff
@@ -56,6 +56,7 @@ if [ "$perltidy_version_found" != "$perltidy_version_expected" ]; then
 fi
 
 find-files() {
+    set +o pipefail # grep might fail
     local files=()
     [[ -d script ]] && files+=(script/*)
     files=($(file --mime-type * "${files[@]}" | grep text/x-perl | awk -F':' '{ print $1 }'))
```

Comment:
Instead of completely disabling pipefails I'd rather add a fallback to the command which can fail:
```
files=($(file --mime-type * "${files[@]}" | (grep text/x-perl || true) | awk -F':' '{ print $1 }'))
```

Reply:
done